ui/src/design/RapiDoc/index.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { RapiDoc } from "./index";
+import exampleSpec from "./example.yaml";
+
+const meta = {
+  title: "Components/RapiDoc",
+  component: RapiDoc,
+  parameters: {
+    layout: "fullscreen",
+  },
+} satisfies Meta<typeof RapiDoc>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    spec: exampleSpec,
+  },
+};
+
+export const DarkTheme: Story = {
```

Comment:
I think we can remove the `DarkTheme` story here. All* of our design components automatically react to light and dark mode, and both themes are automatically documented by the dark mode toggle in storybook. So when using Storybook, every story can be inspected in both themes and don't need an extra dark story.

As far as I see, it is the same here with the Rapidoc component.

*The Code Editor is the only exception, this component needs to get the theme passed via a prop